about.html

```diff
@@ -0,0 +1,118 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WEB3 HACKATHON</title>
+    <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./assets/favicons/favicon-16x16.png">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section id="about-hero">
+        <ul class="top-menu">
+            <li><a href="" class="top-nav-link"><img src="https://img.icons8.com/small/32/FFFFFF/facebook.png"/></a></li>
+            <li><a href="https://twitter.com/blaze_valentine" class="top-nav-link" target="_blank"><img src="https://img.icons8.com/small/32/FFFFFF/twitter.png"/></a></li>
+            <li><a href="" class="top-nav-link">English</a></li>
+            <li><a href="" class="top-nav-link">My Page</a></li>
+            <li><a href="" class="top-nav-link">Logout</a></li>
+        </ul>
+        <div class="hero-section">
+        <nav class="nav-bar">
+            <a href="/" class="logo">
+                <img src="./assets/images/Web3-Hackathon.png" alt="web3 Hackathon Logo">
+            </a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            <ul class="menu">
+                <li><a href="./index.html" class="nav-link">Home</a></li>
+                <li><a href="./#program" class="nav-link">Program</a></li>
+                <li><a href="" class="nav-link">Join</a></li>
+                <li><a href="" class="nav-link">Sponsor</a></li>
+                <li><a href="" class="nav-link">News</a></li>
+                <li><a href="" class="nav-link">CC Campaign</a></li>
+        </nav>
```

Comment:
- [ ] for better semantics, it's better to wrap this section in `header` tag instead of `section` 👍